contracts/BufferLib.sol

```diff
@@ -68,6 +68,7 @@ library BufferLib {
   function seek(Buffer memory _buffer, uint32 _offset, bool _relative) internal pure returns (uint32) {
     // Deal with relative offsets
     if (_relative) {
+      require(_offset + _buffer.cursor > _offset, "Integer overflow when seeking");
```

Comment:
Just for the record: this may be redundant after adding the `notOutOfBounds` modifier from #97

Reply:
Sorry, I double-checked and it turned out `seek` will not be using `notOutOfBounds`.